PoGo.NecroBot.CLI/Config/Translations/translation.id.json

```diff
@@ -18,19 +18,19 @@
     },
     {
       "Key": "wrongAuthType",
-      "Value": "Jenis AuthType pada config.json tidak diketahui"
+      "Value": "Jenis AuthType pada config.json tidak dikenal"
     },
 	{
       "Key": "loginInvalid",
-      "Value": "Kredensial (nama pengguna dan kata kunci) cacat dan gagal masuk"
+      "Value": "Username dan password tidak cocok, login gagal!"
```

Comment:
Yes it is not in bahasa but it is kinda hard (and sounds lame) to put everything into bahasa.